src/Lab/Experiments/SilkBackedSkiaRendering/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Silk.NET.Maths;
+using Silk.NET.Windowing;
+using Silk.NET.Windowing.Glfw;
+using SkiaSharp;
+
+var options = WindowOptions.Default;
+options.Size = new Vector2D<int>(800, 600);
+options.Title = "Silk.NET backed Skia rendering!";
+options.PreferredStencilBufferBits = 8;
+options.PreferredBitDepth = new Vector4D<int>(8, 8, 8, 8);
+GlfwWindowing.Use();
+var window = Window.Create(options);
+window.Initialize();
```

Comment:
Why do we need to use Initialize? We should encourage the use of Load callbacks, this function is only intended for niche use cases.